<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MultiSegWidgetForm</class>
 <widget class="QWidget" name="MultiSegWidgetForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>673</width>
    <height>555</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MultiSeg</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../mseg-ui.qrc">
    <normaloff>:/resources/icon/icon.png</normaloff>:/resources/icon/icon.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_25">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_24">
     <item row="0" column="0">
      <layout class="QGridLayout" name="gridLayout_23">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout_19">
         <item row="0" column="0">
          <layout class="QGridLayout" name="gridLayout_17">
           <item row="0" column="0">
            <widget class="QGroupBox" name="m_inputImageGroupBox">
             <property name="title">
              <string>Input Image</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_2">
              <item row="0" column="0">
               <layout class="QGridLayout" name="gridLayout">
                <item row="0" column="1">
                 <widget class="QPushButton" name="m_inputImagePushButton">
                  <property name="text">
                   <string>Select...</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="0">
                 <widget class="QLineEdit" name="m_inputImageLineEdit">
                  <property name="readOnly">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QGroupBox" name="m_inputBandsGroupBox">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="title">
              <string>Input Bands</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_14">
              <item row="0" column="0">
               <widget class="QListWidget" name="m_inputBandsListWidget">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>50</height>
                 </size>
                </property>
                <property name="selectionMode">
                 <enum>QAbstractItemView::MultiSelection</enum>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <widget class="QGroupBox" name="m_parametersGroupBox">
           <property name="title">
            <string>Parameters</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_5">
            <item row="0" column="0">
             <layout class="QGridLayout" name="gridLayout_4">
              <item row="0" column="0">
               <layout class="QGridLayout" name="gridLayout_6">
                <item row="0" column="0">
                 <widget class="QLabel" name="m_imageTypeLabel">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="text">
                   <string>Image Type:</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QComboBox" name="m_imageTypeComboBox">
                  <item>
                   <property name="text">
                    <string>Radar</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Optical</string>
                   </property>
                  </item>
                 </widget>
                </item>
               </layout>
              </item>
              <item row="1" column="0">
               <layout class="QGridLayout" name="gridLayout_13">
                <item row="0" column="0">
                 <widget class="QLabel" name="m_imageModelLabel">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="text">
                   <string>Image Model:</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QComboBox" name="m_imageModelComboBox">
                  <item>
                   <property name="text">
                    <string>Cartoon</string>
                   </property>
                  </item>
                 </widget>
                </item>
               </layout>
              </item>
              <item row="2" column="0">
               <layout class="QGridLayout" name="gridLayout_3">
                <item row="0" column="0">
                 <widget class="QLabel" name="m_radarImageFormatLabel">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="text">
                   <string>Image Format:</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QComboBox" name="m_radarImageFormatComboBox">
                  <item>
                   <property name="text">
                    <string>Intensity</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Amplitude</string>
                   </property>
                  </item>
                 </widget>
                </item>
               </layout>
              </item>
              <item row="3" column="0">
               <layout class="QGridLayout" name="gridLayout_7">
                <item row="0" column="0">
                 <widget class="QLabel" name="m_levelsLabel">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="text">
                   <string>Levels:</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QSpinBox" name="m_levelsSpinBox">
                  <property name="alignment">
                   <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                  </property>
                  <property name="maximum">
                   <number>100</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item row="4" column="0">
               <layout class="QGridLayout" name="gridLayout_8">
                <item row="0" column="1">
                 <widget class="QDoubleSpinBox" name="m_similarityDoubleSpinBox">
                  <property name="alignment">
                   <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                  </property>
                  <property name="suffix">
                   <string> dB</string>
                  </property>
                  <property name="decimals">
                   <number>1</number>
                  </property>
                  <property name="maximum">
                   <double>10000.000000000000000</double>
                  </property>
                  <property name="singleStep">
                   <double>0.500000000000000</double>
                  </property>
                  <property name="value">
                   <double>1.000000000000000</double>
                  </property>
                 </widget>
                </item>
                <item row="0" column="0">
                 <widget class="QLabel" name="m_similarityLabel">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="text">
                   <string>Similarity:</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item row="5" column="0">
               <layout class="QGridLayout" name="gridLayout_9">
                <item row="0" column="0">
                 <widget class="QLabel" name="m_enlLabel">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="text">
                   <string>ENL:</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QDoubleSpinBox" name="m_enlDoubleSpinBox">
                  <property name="alignment">
                   <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                  </property>
                  <property name="decimals">
                   <number>2</number>
                  </property>
                  <property name="minimum">
                   <double>0.100000000000000</double>
                  </property>
                  <property name="maximum">
                   <double>250.000000000000000</double>
                  </property>
                  <property name="singleStep">
                   <double>1.000000000000000</double>
                  </property>
                  <property name="value">
                   <double>1.000000000000000</double>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item row="6" column="0">
               <layout class="QGridLayout" name="gridLayout_10">
                <item row="0" column="1">
                 <widget class="QComboBox" name="m_confidenceLevelComboBox">
                  <property name="currentIndex">
                   <number>4</number>
                  </property>
                  <item>
                   <property name="text">
                    <string>100%</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>99.9%</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>99.5%</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>99%</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>95%</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>90%</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>85%</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>80%</string>
                   </property>
                  </item>
                 </widget>
                </item>
                <item row="0" column="0">
                 <widget class="QLabel" name="m_confidenceLevelLabel">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="text">
                   <string>Confidence Level:</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item row="7" column="0">
               <layout class="QGridLayout" name="gridLayout_12">
                <item row="0" column="0">
                 <widget class="QLabel" name="m_cvLabel">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="text">
                   <string>Coefficient of Variation:</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QDoubleSpinBox" name="m_cvDoubleSpinBox">
                  <property name="alignment">
                   <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                  </property>
                  <property name="decimals">
                   <number>1</number>
                  </property>
                  <property name="maximum">
                   <double>20.000000000000000</double>
                  </property>
                  <property name="singleStep">
                   <double>0.100000000000000</double>
                  </property>
                  <property name="value">
                   <double>0.600000000000000</double>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item row="8" column="0">
               <layout class="QGridLayout" name="gridLayout_11">
                <item row="0" column="1">
                 <widget class="QSpinBox" name="m_minAreaSpinBox">
                  <property name="alignment">
                   <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                  </property>
                  <property name="suffix">
                   <string> pixels</string>
                  </property>
                  <property name="prefix">
                   <string/>
                  </property>
                  <property name="minimum">
                   <number>1</number>
                  </property>
                  <property name="maximum">
                   <number>10000</number>
                  </property>
                  <property name="value">
                   <number>20</number>
                  </property>
                 </widget>
                </item>
                <item row="0" column="0">
                 <widget class="QLabel" name="m_minAreaLabel">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="text">
                   <string>Minimum Area:</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QGroupBox" name="m_outputDirGroupBox">
           <property name="title">
            <string>Output Directory</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_15">
            <item row="0" column="0">
             <layout class="QGridLayout" name="gridLayout_16">
              <item row="0" column="0">
               <widget class="QLineEdit" name="m_outputDirLineEdit">
                <property name="readOnly">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QPushButton" name="m_outputDirPushButton">
                <property name="text">
                 <string>Select...</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="1">
        <widget class="QGroupBox" name="m_optionsGroupBox">
         <property name="title">
          <string>Options</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_22">
          <item row="0" column="0">
           <layout class="QGridLayout" name="gridLayout_21">
            <item row="0" column="0">
             <layout class="QGridLayout" name="gridLayout_20">
              <item row="0" column="0">
               <widget class="QCheckBox" name="m_enableProgressCheckBox">
                <property name="text">
                 <string>Enable progress interface</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QCheckBox" name="m_savePyramidLevelsCheckBox">
                <property name="text">
                 <string>Save hierarchical pyramid</string>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QCheckBox" name="m_saveIntermediateResultsCheckBox">
                <property name="text">
                 <string>Save intermediate results</string>
                </property>
               </widget>
              </item>
              <item row="3" column="0">
               <widget class="QCheckBox" name="m_resizeIntermediateResultsCheckBox">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>Resize intermediate results to input image size</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="1" column="0">
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <layout class="QGridLayout" name="gridLayout_18">
       <item row="0" column="0">
        <widget class="QPushButton" name="m_helpPushButton">
         <property name="text">
          <string>Help</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="m_okPushButton">
         <property name="text">
          <string>Ok</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QPushButton" name="m_closePushButton">
         <property name="text">
          <string>Close</string>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QPushButton" name="m_aboutPushButton">
         <property name="text">
          <string>About</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>m_inputImagePushButton</tabstop>
  <tabstop>m_inputImageLineEdit</tabstop>
  <tabstop>m_imageTypeComboBox</tabstop>
  <tabstop>m_imageModelComboBox</tabstop>
  <tabstop>m_radarImageFormatComboBox</tabstop>
  <tabstop>m_levelsSpinBox</tabstop>
  <tabstop>m_similarityDoubleSpinBox</tabstop>
  <tabstop>m_enlDoubleSpinBox</tabstop>
  <tabstop>m_confidenceLevelComboBox</tabstop>
  <tabstop>m_cvDoubleSpinBox</tabstop>
  <tabstop>m_minAreaSpinBox</tabstop>
  <tabstop>m_outputDirPushButton</tabstop>
  <tabstop>m_outputDirLineEdit</tabstop>
  <tabstop>m_okPushButton</tabstop>
  <tabstop>m_closePushButton</tabstop>
  <tabstop>m_helpPushButton</tabstop>
 </tabstops>
 <resources>
  <include location="../../mseg-ui.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>m_closePushButton</sender>
   <signal>clicked()</signal>
   <receiver>MultiSegWidgetForm</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>660</x>
     <y>543</y>
    </hint>
    <hint type="destinationlabel">
     <x>191</x>
     <y>4</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_saveIntermediateResultsCheckBox</sender>
   <signal>clicked(bool)</signal>
   <receiver>m_resizeIntermediateResultsCheckBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>453</x>
     <y>90</y>
    </hint>
    <hint type="destinationlabel">
     <x>491</x>
     <y>107</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
